src/index.js

```diff
@@ -103,8 +108,8 @@ export {
   entries,
   clear,
   metaDataCacheName,
-  createCacheName,
-  createCacheEntryName,
+  createMetaCacheName,
+  createMetaCacheEntryName,
```

Comment:
maybe drop the deprecations file and just add wrap the functions with the deprecationNotice here

Reply:
Only issue I'm facing with doing that is either the deprecated method needs to be tested or the deprecation fn needs testing which requires me to export - being internal methods in `src/index.js` would expose them. Any ideas?

I've moved all deprecation to the single file `src/deprecations.js` in the meantime

Reply:
Ideally the deprecated functions should still be tested and tested that when used the deprecation message is given

Reply:
If I dropped the deprecation file in validations, but would still need to test the two functions that create wrappers, otherwise would need to test

Reply:
I've updated as per discussion 1d5424f